assets/shared/images/characters/bf-carOffsets.txt

```diff
@@ -1,4 +1,5 @@
 idle -5 0
+idleHair -5 0
```

Comment:
you're also missing the offset for mom-car

Reply:
> you're missing the idleHair offset for mom-car

mom-car doesn't need it as she have 0 0 idle and not -5 0 like bf. Plus it still puts her into idleHair animation in offset debug menu after her normal idle animation is finished

Reply:
ahh